README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
 
 ## Purpose
 
-To provide an open source, modularized toolbox that enables the pharmaceutical programming community to develop ADaM datasets in R.
+To provide an open source, modularized toolbox that enables the pharmaceutical programming community
+to develop ADaM datasets in R.
 
 ## Installation
 
-Once the package is available from CRAN you'll be able to install it using `install.packages("admiral")`.
+The package is available from CRAN and can be installed by running `install.packages("admiral")`.
 
-In the meantime you can install the latest release of the package directly from GitHub.
+To install the latest development version of the package directly from GitHub use the following code:
 
 ```r
 if (!requireNamespace("remotes", quietly = TRUE)) {
   install.packages("remotes")
 }
-remotes::install_github("pharmaverse/admiral.test", ref = "main") # This is a required dependency of {admiral}
-remotes::install_github("pharmaverse/admiral", ref = "main")
+remotes::install_github("pharmaverse/admiral", ref = "devel")
 ```
 
-## Scope
-
-* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a modular manner (an "opinionated" design strategy)
-* Pharmaceutical communities and companies are encouraged to contribute to `{admiral}` following the provided programming strategy and modular approach
-* All functions are documented, tested, include examples and are listed in the
-[Reference section](https://pharmaverse.github.io/admiral/reference/index.html)
-* Vignettes on how to create ADSL, BDS and OCCUR datasets, including example scripts
-* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation, SMQs ...)
+## Main Goal
 
-## Usage
+Provide data scientists with an open source, modularized toolbox with which to create ADaM datasets
```

Comment:
Can we just say, Provide users or developers?

Reply:
agree - happy to update these mentions all to just say "users"